src/main/java/io/orten/nano/business/UserAPI.java

```diff
@@ -0,0 +1,74 @@
+package io.orten.nano.business;
+
+import io.orten.nano.impl.UserService;
+
+import io.orten.nano.model.User;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.List;
+
+@Path("/")
+public class UserAPI {
+
+    public static List<User> u_list = new ArrayList<User>();

```

Comment:
Switch to using camel case and give it a more meaningful name.
i.e. userList or similar

Reply:
Replaced u_list by userList